### PR TITLE
test: use css var for input label

### PR DESCRIPTION
**Pull request by gustaveen**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4402?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4402** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4402?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4398** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4398?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4396** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4396?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/TextField/TextField.css.ts [hunk 1]
```diff
@@ -42,9 +42,37 @@ export const baseWrapper = style({
...
+  ],
+  large: [
+    baseWrapper,
+    {
+      height: '4.5rem',
+      selectors: {
+        ['&:focus-within, &[data-active=true]']: {
+          vars: {
+            [labelTransform]: `translate(calc(${tokens.space.md} * 0.4), -0.6rem) scale(0.6)`,
+            [labelOverflow]: 'visible',
```
Comment (gustaveen) [inline apps/store/src/components/TextField/TextField.css.ts:70]: To me it feels less obvious to set label styles as a variable in the parent compared to having it inside label

File: apps/store/src/components/TextField/TextField.css.ts [hunk 2]
```diff
@@ -78,23 +106,15 @@ export const inputLabel = styleVariants({
...
     {
       fontSize: tokens.fontSizes.xl,
-      selectors: {
-        [`${baseWrapper}:focus-within > &, ${baseWrapper}[data-active=true] > &`]: {
-          overflow: 'visible',
-          transform: `translate(calc(${tokens.space.md} * 0.4), -0.6rem) scale(0.6)`,
-        },
-      },
+      transform: fallbackVar(labelTransform, 'none'),
+      overflow: fallbackVar(labelOverflow, 'hidden'),
```
Comment (gustaveen) [inline apps/store/src/components/TextField/TextField.css.ts:117]: We would need to add fallbacks to not have undefined styles

File: apps/store/src/components/TextField/TextField.css.ts (hunk 1 above)
Comment (Youakeem) [reply apps/store/src/components/TextField/TextField.css.ts:70]: I see! The point was to make the input `parts` dynamic and reactive to whatever variants we apply to the input wrapper, they themselves don't need to worry about what variants those are, they just use whatever value comes from the variables.

I am ok with not having it this way as well. Go with whatever you feel more intuitive.

Comment (gustaveen) [reply apps/store/src/components/TextField/TextField.css.ts:70]: Cool, thanks! 🙌

**Pull request closed without merging**